.gitignore

```diff
@@ -36,3 +36,4 @@ next-env.d.ts
 
 # contentlayer
 .contentlayer
+public/search.json
```

Comment:
This file keeps getting changed locally, Im not sure if its meant to be checked-in or not but the `git diff` noise is a bit annoying @MaedahBatool